app/components/table.gts

```diff
@@ -220,9 +167,46 @@ export default class Table extends Component<{}> {
     }
   );
 
+  @action
+  updateCustomCellRendering({ api }: { api: GridApi } ) {
+    const cellRenderers = api.getCellRendererInstances();
+    if (!cellRenderers) {
+      return;
+    }
+
+    const emberCellRenderers: EmberCellRenderer[] = [];
+    for (const renderer of cellRenderers) {
+      if (renderer instanceof EmberCellRenderer) {
+        emberCellRenderers.push(renderer);
+      }
+    }
+    this.emberCellRenderers = emberCellRenderers;
+  }
+
+  @action
+  updateCustomTooltipRendering(e: TooltipShowEvent) {
+    if (e.tooltipGui.hasAttribute('data-ember-tooltip') ){
+      this.emberTooltipRenderer = this.context.activeTooltipRender
+    }
+  }
+
   <template>
     <div class="h-96 w-full max-w-7xl mx-auto p-4">
       <div {{this.MountModifier}} class="ag-theme-alpine h-full w-full" />
     </div>
+
+    {{!-- Render ember component from custom cell renderers --}}
+    {{#each this.emberCellRenderers as |renderer|}}
+      {{#in-element renderer.target}}
+        <renderer.component @params={{renderer.params}} />
```

Comment:
Glint/TS doesn't seem to know the types for this.

```bash
The given value does not appear to be usable as a component, modifier, or helper
```

May need to further type-ify things